src/transaction/transaction.rs

```diff
@@ -955,19 +965,20 @@ impl<PdC: PdClient> Committer<PdC> {
     async fn prewrite(&mut self) -> Result<Option<Timestamp>> {
         let primary_lock = self.primary_key.clone().unwrap();
         // FIXME: calculate TTL for big transactions
-        let lock_ttl = DEFAULT_LOCK_TTL;
+        let current_ts = self.rpc.clone().get_timestamp().await?;
```

Comment:
You should store the beginning time in the transaction rather than a new fetching timestamp in prewrite.

Reply:
@TszKitLo40 PTAL

Reply:
done, PTAL